docs/flatcar-linux/packet.md

```diff
@@ -334,7 +334,7 @@ Reference the DNS zone id with `"${aws_route53_zone.zone-for-clusters.zone_id}"`
 | service_cidr | CIDR IPv4 range to assign to Kubernetes services | "10.3.0.0/16" | "10.3.0.0/24" |
 | cluster_domain_suffix | FQDN suffix for Kubernetes services answered by coredns. | "cluster.local" | "k8s.example.com" |
 | enable_reporting | Enable usage or analytics reporting to upstreams (Calico) | false | true |
-| enable_aggregation | Enable the Kubernetes Aggreagation Layer | false | true |
+| enable_aggregation | Enable the Kubernetes Aggreagation Layer | "true" | "false" |
```

Comment:
I'm unsure what is best here. As we don't want the user to set `"true"` as that is a string. Maybe just using `true` (i.e. formatted as code) if that is visialized correctly?

quotes were needed when it was a string. Commit that moved to TF 0.12 removed that on several (https://github.com/kinvolk/lokomotive-kubernetes/commit/75569dc95f5ac95b360d7c0bbcc19284033da266#diff-2668ba9034df1a56b492c1832b871e2eR360) but don't think in all of them :-/

Reply:
We use bools without quotes for other fields, so I consider this out of scope of this PR. Valid point though :)

Reply:
? It's already there, and you already changed it... not sure I understand this answer. Did you change your mind?

Reply:
I think the confusion is because you are talking about an "Oudated" change.
In the current version in master those fields are without the quotes.

Reply:
I see the PR has been updated now. Thanks! :)